cmd/jaeger/internal/all-in-one.yaml

```diff
@@ -33,6 +33,24 @@ extensions:
     #   initial_sampling_probability: 0.1
     http:
     grpc:
+  
+  healthcheckv2:
```

Comment:
Lots of settings here, are they all needed? How are they different from defaults?

Reply:
in all-in-one config we want to have the minimum required configuration.

Reply:
Noted